docker/README.md

```diff
@@ -23,19 +23,44 @@ $ ./run_simulator gazebo-terminus-nvidia gazebo-terminus-nvidia
 $ ./run_simulator gazebo-terminus-intel gazebo-terminus-intel
 ```
 
-The script `run_simulator` will mount the repository folder into the docker container. We recommend working outside the container and placing the files in folder of the repositorys. Be aware that the CONTAINER WILL BE DELETED EACH TIME THAT YOU EXIT IT. This is done in order to make sure that you are not modifying the image that you are working with, so all the colaborators are working with the same image.
+The script `run_simulator` is configured to mount 2 directories:
+
+* Current repository folder (from docker folder, its parent directory)
+* [rndf_gazebo_plugin](https://bitbucket.org/JChoclin/rndf_gazebo_plugin) repository which is supposed to be at the same level of the current repository.
+
+In case the `rndf_gazebo_plugin` repository is not at the same level of the current repository, you should change the mount path in the `run_simulator` script (check `DOCKER_MOUNT_ARGS` variable).
+
+We recommend working outside the container and placing the files in folder of the repositories. Be aware that the CONTAINER WILL BE DELETED EACH TIME THAT YOU EXIT IT. This is done in order to make sure that you are not modifying the image that you are working with, so all the colaborators are working with the same image.
```

Comment:
Beware or be aware? :)

http://english.stackexchange.com/questions/160651/what-is-the-difference-between-beware-and-be-aware